### DISCUSSION TITLE
pasport personalAccessTokensExpireIn now working

### DISCUSSION THREAD

**Discussion by Rasoul-Karimi** [laravel/framework#44602] [General]
hi
I use the last version of laravel (9) and the last version of Passport `laravel/passport": "^10.0`
and in my AuthServiceProvider i use this:

`
        Passport::routes();
        Passport::tokensExpireIn(now()->addDays(30));
        Passport::refreshTokensExpireIn(now()->addDays(50));
        Passport::personalAccessTokensExpireIn(now()->addMonths(6));
`

problem: users' Tokens expire in 1 day! if users login in app, after 1 day there says toneks is expired and need to log in again! 
i want to expire for ex: after 5 Months or 30 day for example!

**Comment by parallels999** [comment]
Last version of passport is `v11.2.1`, not `v10.x`
